dev/TabView/TabView.xaml

```diff
@@ -98,6 +98,7 @@
 
                         <ContentPresenter x:Name="TabContentPresenter"
                             Grid.Row="1"
+                            Background="{ThemeResource TabViewItemHeaderBackgroundSelected}"
```

Comment:
Is this intentional? it looks out of place.

Reply:
Yes, this is so that TabContent and TabViewItem Backgrounds will have the same colour. I'll be going over this detail with design to make sure that's their design intent.